samples/SimpleWebApp/SimpleWebApp.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
```

Comment:
I would change all samples to net7

Reply:
Personally, I would not, as .NET 7 is not a LTS version, but it doesn't matter so much.

However, bumping to .NET 7 should be done in its own PR regardless.